docs/node/timers.md

```diff
@@ -49,9 +49,65 @@ What we mean by "handling jumps": if you had timers that were supposed to go off
 
 You can read more about [Temporal Node SDK's Determinism here](/docs/node/determinism).
 
+## API Examples
+
+`setTimeout` works as normal in JavaScript.
+
+### `sleep`
+
+`sleep` uses the [ms](https://www.npmjs.com/package/ms) package to take either a string or number of milliseconds.
+
+```ts
+/**
+ * Asynchronous sleep. Schedules a timer on the Temporal service.
+ *
+ * @param ms sleep duration - formatted string or number of milliseconds
+ */
+export function sleep(ms: number | string): Promise<void> {
+
+// durably sleep for 30 days
+import { condition, sleep } from '@temporalio/workflow';
+
+await sleep("30 days")
+await sleep(30 * 24 * 60 * 60 * 1000)
+```
+
+### `condition`
+
+`condition` blocks until its given function evaluates to `true`, or optionally, a timeout expires. 
+The timeout also uses the [ms](https://www.npmjs.com/package/ms) package to take either a string or number of milliseconds.
+
+```ts
+/**
+ * Returns a Promise that resolves when `fn` evaluates to `true` or `timeout` expires.
+ *
+ * @param timeout - formatted string or number of milliseconds
+ *
+ * @returns a boolean indicating whether the condition was true before the timeout expires
+ */
+export function condition(timeout: number | string, fn: () => boolean): Promise<boolean>;
+
+/**
+ * Returns a Promise that resolves when `fn` evaluates to `true`.
+ */
+export function condition(fn: () => boolean): Promise<void>;
```

Comment:
I like that we have this in the docs, I'd just rather we didn't duplicate it.
How about making this a snippet?

Reply:
happy to swap it out with a snippet if you add it in the SDK code, but imo the stuff i had to cut out wasn't really worth it